### PR TITLE
Responsive navbar

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
added responsive navigation bar and responsive layout for Component Library. Please review my code.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -78,36 +78,58 @@ const value = arr[1];
...
+const hamburgerButton = document.querySelector("#doc-ham");
+const sidebar = document.querySelector(".doc-sidebar");
+
+hamburgerButton.addEventListener("click",()=>{
+    if(sidebar.classList.contains("sidebar-hidden")){
+        sidebar.classList.remove("sidebar-hidden")
+    }else{
+        sidebar.classList.add("sidebar-hidden")
+
+    }
```
Comment (Ankur9669) [inline app.js]: Suggestion: You can use ternary operator here

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -26,21 +37,57 @@
...
+            "logo ... ... nav-items"
+            "search search search search";
+        padding: 5px 2rem 1rem;
+    }
+}
+
+@media screen and (max-width: 500px) {
+    .navbar-responsive {
+        grid-template-areas:
+            "logo ... ... nav-items"
```
Comment (Ankur9669) [inline components/navigation/navigation.css:81]: fix: @media only screen and max-width

**Review by varun13169 — APPROVED** [review]

**Review by Ankur9669 — APPROVED** [review]
The code looks great to merge

**Review by itsmebipul00 — APPROVED** [review]
Good to merge, clean, nice, work

**Pull request merged**